content/posts/matplotlib/codeswitching-visualization/.ipynb_checkpoints/index-checkpoint.md

```diff
@@ -14,7 +14,7 @@ resources:
       showOnTop: false
 ---
 
-![](Image1.png)
+![ ](Image1.png)
```

Comment:
@isabela-pf please review this alt text.

Is the description enough or should I also add about the twitter handle available on the image? the numeric range of the x and y axis? 

How much information can be considered as sufficient while adding the alt text?

Reply:
I'm summarizing our verbal discussion so other people can read it later!

Because this is the header image on the blog—kind of the preview and summary of the blog post—and because the image is described in more detail throughout the post, I think this is a great description! 

I think the numeric range and axes might be too much information for the header image, and they are described elsewhere so we know that the information is not just absent from the blog post without the image. I'd say the same about the Twitter handle since it is linked at the bottom. 

(If we wanted to keep the handle closer to the image, I'd recommend adding it as a caption like `Plot by WayVSubs2019` because the caption will allow you to link out where alt text will not.)